tests/test_load.py

```diff
@@ -100,9 +100,12 @@ def test_return_type_mo_with_bo():
 #     assert isinstance(mo, se.Model)
 
 # passes
-def test_return_type_nii_with_bo():
-    nii = se.load('example_data', return_type='nii')
-    assert isinstance(nii, se.Nifti)
+
+#TODO: MEMORY ERROR WITH TRAVIS BUILD, MAKE SMALLER BRAIN OBJECT FOR TEST LOAD
+# def test_return_type_nii_with_bo():
+#
+#     nii = se.load('example_data', return_type='nii')
+#     assert isinstance(nii, se.Nifti)
 
```

Comment:
Currently this test is commented out on the CDL repo, just wanted to make sure they were mergeable. I ran this test on my machine (8 GB of memory) and it passes, but it fails on Travis CI. I'm assuming it has to do with the amount of memory the Travis VM uses, as the test also fails in the Docker container, even after allocating just under 7 GB of RAM. 
I would assume it passes on a Docker container with >7 GB of memory, since the test does pass on my machine.

Reply:
This is the same for me, meaning that it passes on my computer but not on Travis.  This test used to pass on Travis, but now fails even with the previous versions.  I think commenting this out makes sense for the time-being and then revisiting the _brain_to_nifti() function to see if we can make it run faster and use less memory by using reshape and/or nilearn masking.